v2/sourcedb-to-spanner/src/main/scripts/create_simple_shard_config.bash

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# This is a sample script to help you generate the shard configuration json for a sharded bulk migration.
+# For the sake of simplicity, the script assumes that:
+# 1. All shards share the same access credentials.
+# 2. All the physical shards share the same names for logical databases.
+#
+# Please modify the script as required for your use case.
+# Dependency: `bash` and `jq`
+
+# Steps to run this script
+# Step-1 Update the variables to match your use case.
+# Step-2 Run the script.
+
+# Variables
+
+# Replace the below with the path in GCS where you would like to upload the file.
+OUTPUT_PATH="gs://bucket/path/to/file.json"
+
+# Replace and add the Ips of all physical shards you would like to migrate in the array below.
+host_ips=(
+"ip.of.first.shard"
+"ip.of.second.shard"
+)
+# Network port of your database, generally 3306 for Mysql.
+port="3306"
+# Replace and add the names of all the logical databases you would like to migrate.
+# Note that all logical databases must have the same schema.
+database_names=("logicalDb01" "logicalDb02")
+# Replace with the actual DB credentials.
+username="<user-name>"
+password="<password>"
+
+# The script below helps you create the shard config json.
+# Main JSON Structure.
+json='{
+  "configType": "dataflow",
+  "shardConfigurationBulk": {
+    "schemaSource": {
+      "dataShardId": "",
+      "host": "",
+      "user": "",
+      "password": "",
+      "port": "",
+      "dbName": ""
+    },
+    "dataShards": []
+  }
+}'
+
+# Code below populates the dataShards array.
```

Comment:
Here also, it helps a new reader if we explicitly state what needs to be filled and what need not be.

Reply:
Same as above. In the steps we have told the user to update the variables and we have a section for the same.